pkg/ranger/grpc_test.go

```diff
@@ -122,13 +124,15 @@ func TestGRPCRangerSize(t *testing.T) {
 		}
 
 		c := client.NewCustomRoute(context.Background(), route)
-		r := GRPCRangerSize(c, "", tt.size)
-		assert.Equal(t, tt.size, r.Size(), errTag)
-		data, err := ioutil.ReadAll(r.Range(tt.offset, tt.length))
+		rr := GRPCRangerSize(c, "", tt.size)
+		assert.Equal(t, tt.size, rr.Size(), errTag)
+		r, err := rr.Range(tt.offset, tt.length)
 		if tt.errString != "" {
 			assert.EqualError(t, err, tt.errString, errTag)
 			return
 		}
+		assert.NoError(t, err, errTag)
+		data, err := ioutil.ReadAll(r)
```

Comment:
here too, we added more functions that return errors, but we didn't actually double the error handling.